addon/globalPlugins/pcKbBrl.py

```diff
@@ -109,7 +110,16 @@
 	"speakDot": "boolean(default=False)",
 	"timeout": "integer(default=0)",
 	"confirmKeys": "string(default=gh)",
-	"cancelKeys": "string(default=ty)"
+	"cancelKeys": "string(default=ty)",
+	"dot1": "string(default="")",
+	"dot2": "string(default="")",
+	"dot3": "string(default="")",
+	"dot4": "string(default="")",
+	"dot5": "string(default="")",
+	"dot6": "string(default="")",
+	"dot7": "string(default="")",
+	"dot8": "string(default="")",
+	"nullKeys": "string(default="")"
```

Comment:
There is a formatting error  in the config spec causing error at NVDA startup when pressing directly NVDA+0 without validating the settings panel first.
Note that it seems that an empty string in config spec should be created by "" (i.e. 2 double quotes) on the opposite to other strings which do not need quotes; instead, you have defined the empty string with no character at all since you have concatenated 2 strings instead (probably not on purpose).
Be careful to double quote including other double quotes.



```suggestion
	"dot1": 'string(default="")',
	"dot2": 'string(default="")',
	"dot3": 'string(default="")',
	"dot4": 'string(default="")',
	"dot5": 'string(default="")',
	"dot6": 'string(default="")',
	"dot7": 'string(default="")',
	"dot8": 'string(default="")',
	"nullKeys'string(default="")',
```

Reply:
Sorry for the syntax error in the suggestion on nullKey line of config spec; seems that you have already fixed it.

Reply:
No problem, I've fixed it. Thanks.